src/status_im/translations/en.cljs

```diff
@@ -438,7 +438,7 @@
    :cost-fee                             "Cost/Fee"
    :nonce                                "Nonce"
    :confirmations                        "Confirmations"
-   :confirmations-helper-text            "Consider transactions complete after 12 confirmations"
+   :confirmations-helper-text            "Transactions with 12 confirmations can be considered irreversible"
```

Comment:
Is it right that transaction with less than 12 confirmation are reversible? The wording seems to imply that.

Reply:
You're right and that's my understanding per this thread. https://ethereum.stackexchange.com/questions/183/how-should-i-handle-blockchain-forks-in-my-dapp/203#203 Though maybe it's not a good idea to let users think there's a way to stop the tx after they've sent it. 

The 12 confirmations comes across as rather specific and we've seen that users think it has something to do with their choices. I'm trying to express that it's related to how Ethereum transactions work (without being too wordy). 

let me try again...

Reply:
Thinking of pulling in a reference to how Ethereum works. WDYT?

`On the Ethereum network, when a transaction has 12 confirmations you can consider it settled.`